syntax = "proto3";
package summa.proto;

import "query.proto";


// Searches documents in the stored indices
service SearchApi {
  // Make search in Summa
  rpc search (SearchRequest) returns (SearchResponse) {}
}
